lib/ansible/module_utils/urls.py

```diff
@@ -622,7 +629,8 @@ def http_request(self, req):
             # close the ssl connection
             #ssl_s.unwrap()
             s.close()
-        except (ssl.SSLError, socket.error), e:
+        except (ssl.SSLError, socket.error):
+            e = sys.exc_info()[1]
```

Comment:
mark this with a comment to revisit with ziploader too -- we'll be able to write some functions to make this more mnemonic and import it into any file we want.